### PR TITLE
Improve cross-linking

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | all

Add some cross-linking between pages and fix broken links

### CODE REVIEW

File: docs/guide/shop_features/shop_features.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ They can also upload a CSV file containing a list of SKUs and quantities.
 
 ## Order history [[% include 'snippets/commerce_badge.md' %]]
 
-When an ERP system is connected, the customer can browse and search past orders, invoices, delivery notes, and credit memos.
+When an ERP system is connected, the customer can browse and search [past orders, invoices, delivery notes, and credit memos](../order_history/order_history.md).
```
Comment (dabrt) [inline docs/guide/shop_features/shop_features.md:34]:
```suggestion
When an ERP system is connected, the customer can browse and search [past orders, invoices, delivery notes, and credit memos](../order_history/orderhistory.md).
```

Also - the title of the target article lacks a space (to be fixed in the mkdocs.yaml file)

**Review by dabrt — APPROVED** [review]

File: docs/guide/shop_features/shop_features.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/shop_features/shop_features.md:34]: Forgot to mention - the file name will be changed in https://github.com/ezsystems/developer-documentation/pull/1331

File: docs/guide/sending_notifications.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ They will appear in their profile in the Back Office.
 
 ## Display notification bars
 
-Notifications are displayed as a message bar in the Back Office.
+[Notifications](sending_notifications.md#display-notification-bars) are displayed as a message bar in the Back Office.
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/sending_notifications.md:17]: I have no idea why I did it...

Comment (DominikaK) [inline docs/guide/sending_notifications.md:17]:
```suggestion
Notifications are displayed as a message bar in the Back Office.
```

**Pull request merged**